website/src/categories/components/dialog/usage.md

```diff
@@ -5,27 +5,154 @@ eleventyNavigation:
 
 </section>
 
 <section>
 
-## How to use
+## Anatomy
+
+<div class="ds-table-wrapper">
+
+| Item | Name | Description | Optional|
+|-|-|-|-|
+| 1 | Overlay | Obscures the page content behind the panel |No|
+| 2 | Panel	| Contains the header, subheader, panel content and actions |No|
+| 3 | Header | An area to display content at the top of the panel (e.g. title)| Yes |
+| 4 | Subheader	| An area to display the description of the title| Yes |
+| 5 | Close button| Icon button for closing the dialog | Yes |
+| 6 | Content | An area to display any content | Yes |
 
-Lorem ipsum dolor sit amet. Consectetur adipiscing elit. Etiam eu turpis molestie, dictum est a, mattis tellus. Sed dignissim, metus nec fringilla accumsan, risus sem sollicitudin lacus, ut interdum tellus elit sed risus. Maecenas eget condimentum velit, sit amet feugiat lectus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Praesent auctor purus luctus enim egestas, ac scelerisque ante pulvinar. Donec ut rhoncus ex. Suspendisse ac rhoncus nisl, eu tempor urna. Curabitur vel bibendum lorem. Morbi convallis convallis diam sit amet lacinia. Aliquam in elementum tellus.
+{.ds-table}
+
+</div>
 
 </section>
 
 <section>
 
-## Content
+## Options
+With these options, you can tweak the appearance of the dialog in Figma. They are available in the Design Panel so you can compose the checkbox to exactly fit the user experience need for the use case you are working on.
+
+<div class="ds-table-wrapper">
+  
+|Item|Options|Description|
+|-|-|-|
+|Device|`'Desktop', 'Mobile'`| Select the device for your prototype to ensure an optimal layout.|
+|Size|`'sm','md' (Default), 'lg', 'xl'`|The dialog come in four sizes: small, medium (default), large and extra large, providing flexibility to align with your design requirements.|
+|Close Icon|`'Yes', 'No'`| Choose if you want to show a close button.|
+|Header Order|`'Title', 'Subtitle'`| Choose if you want to start with the title or subtitle.|
+|Content|`'Yes', 'No'`| Choose if you want to show content (slot component).|
+|Slot Component|`'.slot'`|Select your own component to display within the dialog component.|
+|Actions|`'Yes', 'No'`| Choose if you want to show actions.|
+|Align|`'Left', 'Right'`| Choose if you want to show the actions on the left or right side.|
+
+{.ds-table .ds-table-align-top}
+
+</div>
+  
+</section>
+
+<section>
+  
+## Behavior
+
+### Height
+The height of the dialog is dynamically calculated as a percentage of the available screen. As content is injected into the dialog, it grows vertically to accommodate the information passed to it.
+
+### Entrance and exit motion
+When the user launches the dialog, the overlay fades in from 0% to 100% opacity (transitions) and the dialog slides upward from the centre of the screen (transforms:translate the x or y axis). When the user dismisses the dialog, the same animation occurs in reverse.
```

Comment:
Should we have these detailed descriptions here of animations? What if we change animations in the future? We might forget to update the docs? And this also doesn't describe how the animation migth/will be different on mobile.

Reply:
yeah, good one, will remove it for now